docs/sources/developers/plugins/_index.md

```diff
@@ -28,12 +29,12 @@ You can create several types of plugins, including:
 
 The following topics are covered in this guide:
 
-- **[Get started with plugins]({{< relref "./get-started-with-plugins" >}})** - Start developing Grafana plugins with the [Create-plugin](https://www.npmjs.com/package/@grafana/create-plugin) tool.
-- **[Introduction to plugin development]({{< relref "./introduction-to-plugin-development" >}})** - Learn the fundamentals of Grafana plugin development: frontend and backend development processes, data frames, error handling, and more.
-- **[Create a Grafana plugin]({{< relref "./create-a-grafana-plugin" >}})** - If you're familiar with plugin creation, use the tutorials for creating panel plugins, data source plugins, and more to deepen your knowledge.
+- **[Get started with plugins]({{< relref "./get-started-with-plugins" >}})** - Start developing Grafana plugins with the [create-plugin](https://www.npmjs.com/package/@grafana/create-plugin) tool.
```

Comment:
nit: the ordering doesn't match the ToC and I assume this a conscious ordering.
<img width="1087" alt="Screenshot 2023-06-29 at 17 04 04" src="https://github.com/grafana/grafana/assets/19860021/631f3f45-2cab-4a9b-8c91-039c846e9e3b">

Reply:
Fixed.